public/app/features/dashboard/containers/DashboardPage.test.tsx

```diff
@@ -67,6 +68,7 @@ function dashboardPageScenario(description: string, scenarioFn: (ctx: ScenarioCo
           cancelVariables: jest.fn(),
           templateVarsChangedInUrl: jest.fn(),
           dashboard: null,
+          theme: getTheme(),
```

Comment:
We are moving away from `emotion` to `RTL` so it would be great to convert these tests too.

Reply:
You mean the entire test file?

Reply:
Yes

Reply:
Ok

Reply:
So I started to look at this together with @thisisobate and it's quite an undertaking to migrating this to RTL so we decided to postpone this work.

Reply:
yes. Agreed.